cpp/src/quantiles/quantiles.cu

```diff
@@ -20,60 +20,104 @@
 #include "utilities/cudf_utils.h"
 #include "utilities/error_utils.hpp"
 #include "utilities/type_dispatcher.hpp"
+#include "utilities/wrapper_types.hpp"
 #include "rmm/thrust_rmm_allocator.h"
 #include "cudf.h"
 
 #include <thrust/device_vector.h>
 #include <thrust/copy.h>
 
+QuantiledIndex find_quantile_index(gdf_size_type length, double quant)

```

Comment:
This looks like it should go in the anonymous namespace too.

Reply:
Done for now.